cpp/include/cudf/strings/detail/char_tables.hpp

```diff
@@ -62,6 +89,17 @@ const character_cases_table_type* get_character_cases_table();
  */
 const struct special_case_mapping* get_special_case_mapping_table();
 
+/**
+ * @brief Get the special mapping table index for a given code-point.
+ *
+ * @see cpp/src/strings/char_types/char_cases.h
+ */
+constexpr uint16_t get_special_case_hash_index(uint32_t code_point)
+{
+  constexpr uint16_t special_case_prime = 499;  // computed from generate_special_mapping_hash_table
```

Comment:
It is recommended to use UPPER_CASE for `constexpr` values if possible.

Reply:
I don't see the recommendation documented. It appears this assertion is not true for most (>1000) cases in libcudf.

Reply:
It's not officially recommended in cudf. I vaguely remember that I've read that convention somewhere else and tried to follow it closely. Anway, it's just an unofficial recommendation.